src/app/portlets/dot-edit-content/services/html/iframe-edit-mode.js.ts

```diff
@@ -1,7 +1,27 @@
 export const GOOGLE_FONTS = 'https://fonts.googleapis.com/css?family=Roboto:400,700';
+export const MODEL_VAR_NAME = 'dotNgModel';
+
 export const EDIT_PAGE_JS = `
 (function () {
     var containers = Array.from(document.querySelectorAll('div[data-dot-object="container"]'));
+
+    function getDotNgModel() {
+        var model = [];
+        containers.forEach(function(container) {
+            var dotContainerColumn = {};
+            dotContainerColumn.id = container.dataset.dotIdentifier;
+            dotContainerColumn.uuid = container.dataset.dotUuid;
+            var contentlets = Array.from(container.querySelectorAll('div[data-dot-object="contentlet"]'));
+
+            dotContainerColumn.contentletsId = contentlets.map(function(contentlet) {
```

Comment:
I think that ```contentlets``` will do the trick... it's kinda how it comes from the server with containers.